elessar/pages/Login/store.tsx

```diff
@@ -0,0 +1,27 @@
+import { AsyncStorage } from 'react-native';
+
+
+export async function storeData(key: string, data: any) {
+    try {
+      await AsyncStorage.setItem(
+        key,
+        data
+      );
+    } catch (error) {
+      console.log('Store Item problem')
+      console.log(error)
+    }
+  };
+
+export async function retrieveData (key: string){
+    try {
+      const value = await AsyncStorage.getItem(key);
+      if (value !== null) {
+        // We have data!!
+        return value;
+      }
```

Comment:
Aqui se o valor for `null` essa função acaba virando um void. Talvez valha a pena vc retornar de qualquer forma, e depois quando precisar do token pra alguma coisa vc verifica se ele tme conteúdo.